README.md

```diff
@@ -95,6 +90,15 @@ elasticsearch:
     region: ''
 ```
 
+### Sentry logging
+
+Server side logging via [Sentry](http://sentry.io) can be enabled using the following configuration:
```

Comment:
Dunno if this is important but should there be contact info regarding how to access the logs as they are password protected?

Reply:
Guessing probably not....

Reply:
I think @Koded wanted to keep all keys out of the repo (which I already committed once and then removed 😬 )

Reply:
Fully - just wondered if there needed to be a named contact person to administer this - but probs not : )))